smartsim/settings/launchSettings.py

```diff
@@ -58,18 +62,18 @@ def __init__(
             self._launcher = LauncherType(launcher)
         except ValueError:
             raise ValueError(f"Invalid launcher type: {launcher}")
-        self._arg_builder = self._get_arg_builder(launch_args)
+        self._arguments = self._get_arguments(launch_args)
         self.env_vars = env_vars or {}
 
     @property
     def launcher(self) -> str:
-        """Return the launcher name."""
+        """The launcher type"""
```

Comment:
:returns: comment?

Reply:
Good catch!

Reply:
actually let me take a look at how sphinx formats properties. If it treats them like attrs it may not be expecting a `:returns:` block

Reply:
okay, so we can add an `:returns:` block to a `@property`, but the resulting output (as we currently have sphinx configured to format it) would look something like this:

![Screenshot 2024-07-23 at 4 56 54 PM](https://github.com/user-attachments/assets/21165c6b-7cae-42ae-8cb7-0e5b14630c5d)

Still willing to add but I'm a bit worried that this could be end up becoming slightly redundant
```py
    @property
    def launcher(self) -> str:
        """Return the launcher name.
        
        :returns: the launcher name.
        """
        ...
```
Thoughts?

Reply:
Decided to add `:returns:` tags to `@property`s just in case any created in the future have a side effect in future. It was decided that consistency with those cases is more important than eliminating redundancy.